browser/js/views/summary/summary.html

```diff
@@ -0,0 +1,72 @@
+<!doctype html>
+<html>
+    <head> 
+    </head>
+    <body>
+        <div class="content">
+            
+            <div class="classHeader">Promises and Arrays</div>
+            <div class="classInstructor">Omri Bernstein</div>
+
+            <div class="video"></div>
+            <div class="chart"></div>
+            <div class="questionQueue"></div>
+            <div class="pollQueue"></div>
+
+        </div>
+        
+        <style>
```

Comment:
Do you mind putting your styling in an SCSS file such as "browser/scss/summary/main.scss"? if you run "gulp buildCSS" it will add the code in your summary/main.scss to the browser/scss/main.scss file.